src/util/calculateMargin.ts

```diff
@@ -0,0 +1,22 @@
+import { DROPEND_MARGINLEFT, MARGINLEFT_CHANGE } from '../constants'
+
+/**
+ * Calculates the change in the margin-left of drop-end to compensate for the shift towards the right on mobile devices.
+ */
+function calculateNewMargin() {
```

Comment:
This function always returns `MARGINLEFT_CHANGE`...?

Reply:
Yes, it returns the updated change in drop-end margin-left, maybe instead of making it global I should move it inside the function since we don't really need it anywhere else.

Reply:
No, I mean it always returns `2.5`, no matter what.